examples/kafka/kafka_flow.py

```diff
@@ -0,0 +1,56 @@
+import time
+from prefect.tasks.kafka.kafka import KafkaBatchConsume, KafkaBatchProduce
+from prefect import task, Flow
+
+TOPIC = "example_events"
+BOOTSTRAP_SERVER = "localhost:9092"
+GROUP_ID = "1"
+
+
+@task
+def print_results(x):
+    print(f"First two messages: {x[:2]}")
+    print(f"Last two messages: {x[-2:]}")
+    print(f"Total messages: {len(x)}")
+
+
+kafka_consume = KafkaBatchConsume(BOOTSTRAP_SERVER, GROUP_ID)
+kafka_produce = KafkaBatchProduce(BOOTSTRAP_SERVER)
+
+
+with Flow("Kafka Example") as flow:
+
+    messages = [{"key": str(i), "value": str(i)} for i in range(30000)]
+
+    kafka_produce.run(
+        topic=TOPIC,
+        messages=messages[0:20000],
+        flush_threshold=1000,
+    )
+
+    kafka_produce.run(
+        bootstrap_servers=BOOTSTRAP_SERVER,
+        topic=TOPIC,
+        messages=messages[20000:],
+        flush_threshold=1000,
+    )
+
+    time.sleep(5)
```

Comment:
instead of `time.sleep`, can you set the produce upstream of the consume tasks? the docs here may be helpful https://docs.prefect.io/core/concepts/flows.html#imperative-api